CMakeLists.txt

```diff
@@ -559,12 +559,13 @@ include_directories(${CONFIG_INCLUDE_PATH})
 include (cmake/warnings.cmake)
 include (cmake/print_flags.cmake)
 
+add_subdirectory (base)
+add_subdirectory (src)
+
```

Comment:
but master compiles just fine... I don't see how initial issue you were fixing could possibly trigger this error

Reply:
Try linking with gnu ld, this will appear. on s390x, its' the only linker available.